index.js

```diff
@@ -61,7 +65,7 @@ exports.searchFirestore = functions.firestore
                 }
                 querySnapshot.forEach(function (docSnapshot) {
                     var snapshotData = docSnapshot.data();
-                    documents[docSnapshot.id] = docSnapshot.data();
+                    documents[docSnapshot.id] = snapshotData;
```

Comment:
@horaciocome1 This was an intentional duplicate call. `snapshotData` receives a `key` attribute on the line bellow so that it can be added to `lunr`. Also notice `snapshotData` is different from `docSnapshot.data()` added to the `documents` array, as this latter doesn't need the `key` attribute.

Reply:
Thanks.
Got it and made the changes.
Just waiting for your review now.